src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -37,4 +51,8 @@ public String getName() {
     public String getColor() {
         return color;
     }
+
+    public List<Section> getSections() {
+        return sections;
```

Comment:
외부에 참조가 그대로 노출되고 있는데, 변경되지 않도록 아래와 같이 반환하는건 어떨까요?

```
```suggestion
    public List<Section> getSections() {
        return Collections.unmodifiableList(sections);
```

Reply:
피드백 반영하면서 불필요해져서 제거했습니다!